src/components/KToaster/ToastManager.ts

```diff
@@ -67,7 +67,7 @@ export default class ToastManager {
     })
   }
 
-  close(key: any): void {
+  close(key: string): void {
```

Comment:
question: the interface is changing here; is that going to create a downstream issue?